ext/bin/pdb-dataset

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+usage() { echo 'Usage: PDBBOX=BOX_DIR pdb-dataset import --timeshift-to now DUMPFILE'; }
+
+if test "$#" -lt 1 -o -z "$PDBBOX"; then
+  usage 1>&2
+  exit 1
+fi
+
+misuse()
+{
+  usage 1>&2
+  exit 1
+}
+
+timeshift=''
+timeshiftsql=''
+dumpfile=''
+command=''
```

Comment:
I'd suggest checking for the command first, outside the while loop, i.e. requiring for now that the subcommand to be the first argument.  This prevents any ambiguity about whether an argument is a command or a positional argument (like a datafile).

For example (assuming the error checking of command later is sufficient), perhaps just:
```
test $# -gt 0 || misuse
command="$1"
shift 1
```
And accordingly, I'd imagined that the other arguments like `--timeshift-to` and `DUMPFILE` are `import` specific arguments, not a general args, so I might move the while loop into the `import)` handling lower down.
  
i.e. we don't have any "global" options right now (like git's `--git-dir`) that might go before the subcommand.

Though since we only have one subcommand right now, it's not critical; we can always move the loop into the `import)` section later if you prefer.

Reply:
this makes sense. moved everything under the `import)` handling